students/Sean_Tasaki/Lesson08/test_circle.py

```diff
@@ -0,0 +1,78 @@
+
+
+import pytest
+from circle import *
```

Comment:
ideally you would import specific functions from circle, or simply import circle